docs/api/web-contents.md

```diff
@@ -29,19 +29,19 @@ console.log(webContents)
 
 ### `webContents.getAllWebContents()`
 
-Returns an array of all `WebContents` instances. This will contain web contents
+Returns `WebContents[]` - An array of all `WebContents` instances. This will contain web contents
```

Comment:
Returns an array of WebContents instances which correspond to the contents of all windows, webviews, opened devtools, and devtools extension background pages.